src/angular/hq/src/app/staff-dashboard/staff-dashboard.component.html

```diff
@@ -1,7 +1,40 @@
 <div class="px-5">
-  <div class="flex items-center justify-between gap-[20px]">
-    <div class="text-4xl text-white">
-      {{ (staffDashboardService.time$ | async)?.staffName || "-" }}
+  <div class="flex justify-between gap-[20px]">
+    <div class="flex flex-col justify-between">
+      <div class="text-4xl text-white">
+        {{ (staffDashboardService.time$ | async)?.staffName || "-" }}
+      </div>
+      <div class="grid relative w-full max-w-[220px]">
+        <select
```

Comment:
After setting status successfully, show a toast notification so the user knows something happened.